
CollvoidLocalPlanner:
  holo_robot: false
  wheel_base: 0.25
  
  max_vel_x: 0.5
  max_vel_th: 1.5
  min_vel_x: 0.1
  min_vel_th: 0.2
  min_vel_y: 0.0
  max_vel_y: 0.0
  min_vel_th_inplace: 0.5
  acc_lim_x: 5.0
  acc_lim_y: 5.0
  acc_lim_th: 5.2

  max_vel_with_obstacles: 0.5

  footprint_radius: 0.20 #was 0.17

TrajectoryPlannerROS:
  max_vel_x: 0.50
  min_vel_x: 0.10
  max_rotational_vel: 1.5
  min_in_place_rotational_vel: 0.1
  acc_lim_th: 0.75
  acc_lim_x: 0.50
  acc_lim_y: 0.50
  holonomic_robot: false
  yaw_goal_tolerance: 0.05
  xy_goal_tolerance: 0.1
  goal_distance_bias: 0.8
  path_distance_bias: 0.6
  sim_time: 1.5
  heading_lookahead: 0.325
  oscillation_reset_dist: 0.05
  vx_samples: 6
  vtheta_samples: 20
  dwa: false


  
local_costmap:
  global_frame: odom
  robot_base_frame: base_link
  update_frequency: 5.0
  publish_frequency: 5.0
  static_map: false
  rolling_window: true
  width: 4.0
  height: 4.0
  resolution: 0.025
  transform_tolerance: 0.5

  map_type: voxel
  obstacle_range: 2.5
  raytrace_range: 3.0
  robot_radius: 0.19
  #footprint: [[0.21, 0.21], [-0.21, 0.21], [-0.21, -0.21], [0.21, -0.21]]
  inflation_radius: 0.20
  footprint_padding: 0.01

  observation_sources: laser_scan_sensor 
  laser_scan_sensor: {sensor_frame: base_link, data_type: LaserScan, topic: base_scan, marking: true, clearing: true}

global_costmap:
  global_frame: /map
  robot_base_frame: base_link
  update_frequency: 3.0
  publish_frequency: 0.0

  static_map: false
  rolling_window: true
  width: 100.0
  height: 100.0
  resolution: 0.05

  map_type: costmap

  obstacle_range: 2.5
  raytrace_range: 3.0
  robot_radius: 0.18
  #footprint: [[0.21, 0.21], [-0.21, 0.21], [-0.21, -0.21], [0.21, -0.21]]
  inflation_radius: 0.22
  footprint_padding: 0.01

  
  #robot_radius: 0.18
  # footprint: [[0.17, 0.17], [-0.17, 0.17], [-0.17, -0.17], [0.17, -0.17]]
  #inflation_radius: 0.20
  #footprint_padding: 0.01

  observation_sources: laser_scan_sensor 
  laser_scan_sensor: {sensor_frame: base_link, data_type: LaserScan, topic: base_scan, marking: true, clearing: true}

